.github/workflows/ci.yml

```diff
@@ -460,50 +391,46 @@ jobs:
                                       --binary="$DENO_BIN"          \
                                       --json=wpt.json               \
                                       --wptreport=wptreport.json
-
       - name: Upload wpt results to dl.deno.land
         continue-on-error: true
-        if: |
+        if: |-
           runner.os == 'Linux' &&
           matrix.job == 'test' &&
           matrix.profile == 'release' &&
           github.repository == 'denoland/deno' &&
           github.ref == 'refs/heads/main' && !startsWith(github.ref, 'refs/tags/')
-        run: |
+        run: |-
           gzip ./wptreport.json
           gsutil -h "Cache-Control: public, max-age=3600" cp ./wpt.json gs://dl.deno.land/wpt/$(git rev-parse HEAD).json
           gsutil -h "Cache-Control: public, max-age=3600" cp ./wptreport.json.gz gs://dl.deno.land/wpt/$(git rev-parse HEAD)-wptreport.json.gz
           echo $(git rev-parse HEAD) > wpt-latest.txt
           gsutil -h "Cache-Control: no-cache" cp wpt-latest.txt gs://dl.deno.land/wpt-latest.txt
-
       - name: Upload wpt results to wpt.fyi
         continue-on-error: true
-        if: |
+        if: |-
           runner.os == 'Linux' &&
           matrix.job == 'test' &&
           matrix.profile == 'release' &&
           github.repository == 'denoland/deno' &&
           github.ref == 'refs/heads/main' && !startsWith(github.ref, 'refs/tags/')
         env:
           WPT_FYI_USER: deno
-          WPT_FYI_PW: ${{ secrets.WPT_FYI_PW }}
-          GITHUB_TOKEN: ${{ secrets.DENOBOT_PAT }}
-        run: |
+          WPT_FYI_PW: '${{ secrets.WPT_FYI_PW }}'
```

Comment:
I'm not so familiar with YAML, but in this situation both sides are equivalent? Adding quotes doesn't change how it's evaluated?

Reply:
`'${{ matrix.os }}'`, `name: '${{ matrix.job }} ${{ matrix.profile }} ${{ matrix.os }}'`, etc. seem to work above, so I assume this works too.